src/server/protocol.ts

```diff
@@ -1232,6 +1232,8 @@ namespace ts.server.protocol {
          */
         formatOptions?: FormatCodeSettings;
 
+        servicesOptions?: ServicesSettings;

```

Comment:
consider renaming this to just `options`.

Reply:
Should the type be `Options` too?

Reply:
yeah.